modules/react/form-field/lib/hooks/useFormFieldModel.tsx

```diff
@@ -25,7 +25,7 @@ export const useFormFieldModel = createModelHook({
      */
     isRequired: false,
 
-    orientation: 'vertical' as 'vertical' | 'horizontal' | 'horizontalEnd' | 'horizontalStart',
+    orientation: 'vertical' as 'vertical' | 'horizontalEnd' | 'horizontalStart',
```

Comment:
Question: Why `'vertical' as 'vertical'`?

Reply:
I think this types the default value of `vertical` as the following string union.